lib/src/test/java/uk/gov/di/ipv/cri/kbv/api/util/KBVAnswerStorageMapperTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.di.ipv.cri.kbv.api.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.cri.kbv.api.domain.KBVAnswerItem;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuestion;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuestionOptions;
+import uk.gov.di.ipv.cri.kbv.api.domain.QuestionsResponse;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class KBVAnswerStorageMapperTest {
+    @Test
+    void shouldMapQuestionResponseQuestionOptionsToKBVAnswerItems() {
+        QuestionsResponse questionsResponse =
+                getExperianQuestionResponse(new KbvQuestion[] {getQuestionOne(), getQuestionTwo()});
+        KBVAnswerStorageMapper kbvAnswerStorageMapper = new KBVAnswerStorageMapper();
+        List<KBVAnswerItem> kbvAnswerServices =
+                kbvAnswerStorageMapper.mapToKBVAnswerItems(questionsResponse);
+
+        assertAll(
+                () -> assertEquals("Q00015", kbvAnswerServices.get(0).getQuestionId()),
+                () -> assertEquals("UP TO £10,000", kbvAnswerServices.get(0).getAnswer()),
```

Comment:
It might be worth referring to the TestDataCreator to get questionOne / getQuestionTwo 
& then refer to the options collection rather than hard-coding the strings here